OWML.Common/MessageType.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.Common
+{
+    public enum MessageType
```

Comment:
LogType?

Reply:
Actually, maybe message type makes sense - considering my comment about the magic quit phrase...